### ISSUE TITLE
Support for Spark 3.4.x

### ISSUE THREAD

**Issue by aymane-elbouchibti** [microsoft/sql-spark-connector#251]
Hello, 

Can we get an estimated date for the release of the version 1.4.0 (which support spark 3.4.x) in GA ?

Thank you.

**Comment by lbcommer** [comment]
Same question here. That version has been in beta since September and is not even included in Maven.

**Comment by atul-delphix** [comment]
Yes, please update here, waiting from long time and all the current version has lots of security vulnerabilities.
